addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5433,7 +5442,7 @@ msgstr ""
 
 #: xbmc\osx\XBMCHelper.cpp
 msgctxt "#13145"
-msgid "If you are currently using the Apple Remote to control"
+msgid "If you are using the Apple Remote to control"
```

Comment:
The warning is for people who are changing the setting when actually using the Apple remote, so that they know they would have to switch to something else to re-enable that setting (should they actually turn it off). If someone is changing a setting using a keyboard or smart phone remote then they are not at risk of losing control at that very moment.